packages/orchestration/src/cosmos-api.ts

```diff
@@ -212,12 +212,18 @@ export interface IcaAccount {
     msgs: AnyJson[],
     opts?: Partial<Omit<TxBody, 'messages'>>,
   ) => Promise<string>;
-  /** get Purse for a brand to .withdraw() a Payment from the account */
-  getPurse: (brand: Brand) => Promise<Purse>;
   /**
-   * Close the remote account
+   * Closes the ICA channel (account). Does not retrieve assets, so warn the
+   * caller to retrieve them first. However, the Port is persisted and
```

Comment:
"so warn the caller" should should this? Isn't this comment the thing that should?

Worth calling out more loudly: 
```
CAVEAT: Does not retrieve assets so they may be lost if left.
```